### ISSUE TITLE
CustomImportOrder: introduce a new message for the violation within import group

### ISSUE THREAD

**Issue by gmshake** [checkstyle/checkstyle#7278]
https://checkstyle.org/config_imports.html#CustomImportOrder

```
/var/tmp $ javac CustomImportOrder.java
```

```
/var/tmp $ cat config.xml
```
```xml
<!DOCTYPE module PUBLIC "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
 "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name="Checker">
  <module name="TreeWalker">

    <module name="CustomImportOrder">
      <property name="customImportOrderRules" value="STANDARD_JAVA_PACKAGE"/>
    </module>

  </module>
</module>
```

```
/var/tmp $ cat CustomImportOrder.java
```
```java
package com.example;

import java.io.File;


import java.util.Arrays;

class CustomImportOrder {
}
```

```
/var/tmp $ RUN_LOCALE="-Duser.language=en -Duser.country=US"
/var/tmp $ java $RUN_LOCALE -jar checkstyle-8.26-all.jar \
    -c config.xml CustomImportOrder.java
Starting audit...
[ERROR] /var/tmp/CustomImportOrder.java:6: 'java.util.Arrays' should be separated from previous import group by one line. [CustomImportOrder]
Audit done.
Checkstyle ends with 1 errors.
```

---------------

The violation message is misleading as it is the same as the message between different import groups.

I'd like to propose that we introduce a new message for the violation within import group, some one like ```ImportOrderCheck#MSG_SEPARATED_IN_GROUP```.

--------------

**Comment by romani** [comment]
Issue is approved

**Comment by rnveach** [comment]
Fix was merged

**Comment by romani** [comment]
as metadata of Check is changed ... it is new feature to let plugins clearly see that they need to be updated.
